cellpack/autopack/Analysis.py

```diff
@@ -1538,70 +1717,79 @@ def doloop(
+        angle_file = self.env.out_folder / f"angles_{packing_basename}.txt"
+        position_file = self.env.out_folder / f"positions_{packing_basename}.txt"
+        all_pos_file = self.env.out_folder / f"all_positions_{packing_basename}.json"
+        center_distance_file = (
+            self.env.out_folder / f"center_distances_{packing_basename}.txt"
+        )
+        ingredient_distance_file = (
+            self.env.out_folder / f"ingredient_distances_{packing_basename}.txt"
+        )
+        occurences_file = self.env.out_folder / f"occurence_{packing_basename}.json"
         rangeseed = range(num_seeds)
-        distances = {}
-        ingredient_positions = {}
-        ingredient_angles = {}
+        center_distances_dict = {}
+        ingredient_distances_dict = {}
+        ingredient_positions_dict = {}
+        ingredient_angles_dict = {}
         occurences = {}
-        all_pos_dict = {}
+        all_positions_dict = {}
         total_positions = []
-        total_distances = []
+        total_center_distances = []
         total_angles = []
-        angles = None
         rebuild = True
+
         for seed_index in range(num_seeds):
-            basename = f"seed{seed_index}_{basename_all}"
-            self.env.result_file = (
-                f"{self.env.out_folder}/seed_{seed_index}_{basename_all}"
+            seed_basename = f"seed{seed_index}_{packing_basename}"
+            self.env.result_file = str(
+                self.env.out_folder / f"seed_{seed_index}_{packing_basename}"
             )
             # Clear
             if self.afviewer:
                 self.afviewer.clearFill("Test_Spheres2D")
             else:
                 self.env.reset()
             self.env.saveResult = True
-            seed = seeds_i[seed_index]  # int(time())
+            seed = seed_list[seed_index]  # int(time())
             self.build_grid(
                 forceBuild=rebuild,
             )
             two_d = self.env.is_two_d()
+            use_simularium = False
             self.pack(
                 seed=seed,
                 vTestid=seed_index,
                 vAnalysis=0,
                 fbox_bb=fbox_bb,
-                show_plotly_plot=(show_grid and two_d),
+                # TODO: fix this to disable plotly if using simularium
+                show_plotly_plot=(show_grid and two_d) and not use_simularium,
             )
             self.center = self.env.grid.getCenter()
             if render:
                 # render/save scene if hosted otherwise nothing
-                self.helper.render(basename + ".jpg", 640, 480)
-                self.helper.write(basename + ".c4d", [])
+                self.helper.render(seed_basename + ".jpg", 640, 480)
+                self.helper.write(seed_basename + ".c4d", [])
 
             if plot and two_d:
-                width = 1000.0  # should be the boundary here ?
-                fig = pyplot.figure()
+                width = self.env.get_size_of_bounding_box()
```

Comment:
oh, this was causing problems before, thanks for fixing

Reply:
Probably doesn't fit the scope of this PR, but it's a small enough change that we can probably keep it.